tests/providers/class-two-factor-backup-codes-ajax.php

```diff
@@ -1,110 +0,0 @@
-<?php
```

Comment:
Using `git mv` to rename this would reduce the diff noise and make it clear what changes were needed to this file.